packages/server/src/environment.ts

```diff
@@ -3,28 +3,8 @@ import { ServiceType } from "@budibase/types"
 coreEnv._set("SERVICE_TYPE", ServiceType.APPS)
 import { join } from "path"
 
-function isTest() {
-  return isCypress() || isJest()
-}
-
-function isJest() {
```

Comment:
The `isJest` is likely still needed - I don't think we can remove all the coverage around checking if it is in test.